docs/home/howto/MONITORING.md

```diff
@@ -325,9 +325,15 @@ specification:
   - kibanaserver
 ```
 
-### Upgrade of Elasticsearch, Kibana and Filebeat
+### Upgrade of OpenSearch, OpenSearch Dashboards and Filebeat
+
+Keep in mind taht during the upgrade process Epiphany takes `kibanaserver` (for Dashboards) and `logstash` (for Filebeat) user passwords and re-applies them to upgraded configuration of Filebeat and Kibana. So if these password phrases differ from what was setup before upgrade, you should reflect these changes upon next login process.
+
+ Epiphany upgrade of OpenSearch, OpenSearch Dashboards or Filebeat components will fail if `kibanaserver` or `logstash` usernames were changed in configuration of OpenSearch, OpenSearch Dashboards or Filebeat before.
+
+<br>
 
-During upgrade Epiphany takes `kibanaserver` (for Kibana) and `logstash` (for Filebeat) user passwords and re-applies them to upgraded configuration of Filebeat and Kibana. Epiphany upgrade of Open Distro, Kibana or Filebeat will fail if `kibanaserver` or `logstash` usernames were changed in configuration of Kibana, Filebeat or Open Distro for Elasticsearch.
+<sup>[1] For the backward compatibility needs, some naming conventions ( ie. kibanaserver user name ) are still present within the new ( OpenSearch ) platform though they will be suppresed in the future. In aftermath, Epiphany stack is also still using these names.</sup>
```

Comment:
Error with tags? **\<sup\>\[1\]**...

Reply:
Nope. Superscript was aimed here as this is a lower importance notice.